lib/components/common/Info.tsx

```diff
@@ -108,17 +108,18 @@ const styles = {
     }
 };
 
-const Info = (props: { setIsInfoOpen: (b: boolean) => void }) => {
-    const {setIsInfoOpen} = props
+const Info = (props: { setIsInfoOpen: (b: boolean) => void, onAfterSidebarToggle?: () => void }) => {
```

Comment:
issue (blocking): These props can be collapsed into one prop, named for example, onClose